server/api/submission/submission.integration.js

```diff
@@ -31,27 +90,35 @@ describe('Submission API:', function() {
   });
 
   describe('POST /api/submissions', function() {
+
+
     beforeEach(function(done) {
       request(app)
-        .post('/api/submissions')
-        .send({
-          title: 'New Submission',
-          content: 'This is the brand new submission!!!'
-        })
-        .expect(201)
-        .expect('Content-Type', /json/)
-        .end(function(err, res) {
-          if (err) {
-            return done(err);
-          }
-          newSubmission = res.body;
-          done();
-        });
+      .post('/api/submissions')
+      .set('authorization', 'Bearer ' + token)
+      .field('Content-Type', 'multipart/form-data')
+      .field('userId', '000000000000000000000004')
+      .field('authors[0]', '000000000000000000000004')
+      .field('authors[1]', '000000000000000000000005')
+      .field('eventId', '000000000000000000000020')
```

Comment:
:penguin: we might want to define these user ids in a test config file just because it's really hard to tell what is being tested see #124